webots_ros2_tiago/resource/ros2_control.yml

```diff
@@ -13,8 +13,9 @@ diffdrive_controller:
     left_wheel_names: ["wheel_left_joint"]
     right_wheel_names: ["wheel_right_joint"]
 
-    wheel_separation: 0.5
-    wheel_radius: 0.12
+    # Values don't correspond exactly to the ones in the robot URDF but result in more accurate odometry
```

Comment:
For this change I think you should use these parameters:
```yaml
wheel_separation_multiplier: 0.88
left_wheel_radius_multiplier: 0.8167
right_wheel_radius_multiplier: 0.8167
```
The wheel radius and the  wheel separation are physical parameters which should be the same as in the URDF.

Reply:
Thank you. @ygoumaz is going to investigate this soon.